components/lib/createArComponent.js

```diff
@@ -69,20 +69,19 @@ export default (mountConfig, propTypes = {}, nonUpdateablePropKeys = []) => {
   // any custom props (material, shape, ...)
   const nonNodePropKeys = keys(propTypes);
 
-  const processColors = props => ({
+  const parseMaterials = props => ({
```

Comment:
there is a file called "parseMaterial", but exported functions have completly different names.

But here you have function "parseMaterials". I know it was already confusing to begin with, but maybe we can improve here

Reply:
I renamed it to `processMaterial` to match the `processColor` function we are already using